src/App.css

```diff
@@ -1,38 +1,572 @@
+  width: 60%;
+  display: table;
+  background-color: rgb(238, 238, 238);
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  border-radius: 10px;
+}
+@media (max-width: 750px) {
+  .masterVolume {
+    width: 99%;
+  }
+}
+
+/* links */
+/* unvisited link */
+a:link {
+  color: black;
+  border-bottom: 1px dotted #000;
+  text-decoration: none;
+}
+
+/* visited link */
+a:visited {
+  color: black;
+  border-bottom: 1px dotted #000;
+  text-decoration: none;
+}
+
+/* mouse over link */
+a:hover {
+  color: #454adb;
+  border-bottom: 1px dotted #000;
+  text-decoration: none;
+}
+
+/* selected link */
+a:active {
+  color: #c870f1;
+  border-bottom: 1px dotted #000;
+  text-decoration: none;
+}
+
+/* buttons */
+
+button {
+  background-color: #e6e4e3;
+  border: 1px inset;
+  border-color: black;
+  color: black;
+  padding: 10px 16px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 14px;
+  border-radius: 10px;
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  /* disable highlight */
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+
+/* most buttons */
+.mb:active {
+  background-color: #e0b3f5;
+}
+
+.fxButtons:active {
+  background-color: #9294da;
+}
+
+button:active {
+  transform: translateY(1px);
+  z-index: 1;
+}
+
+button:focus {
+  outline: none;
+}
+
+.infoButton {
+  padding: 4px 4px;
+}
+.infoButton:active {
+  background-color: #e0b3f5;
+}
+
+.regularPlayButton {
+  background-color: #e6e4e3;
+}
+
+.activeButton {
+  background-color: #e0b3f5;
+  transform: translateY(1px);
+  z-index: 1;
+}
+
+.rootButton {
+  background-color: #f5deb3;
+}
+
+.activeRootButton {
+  background-color: #c870f1;
+  transform: translateY(1px);
+  z-index: 1;
+}
+
+@media (max-width: 1500px) {
+  .regularPlayButton,
+  .activeButton,
+  .rootButton,
+  .activeRootButton {
+    width: fit-content;
+    padding: 10px 5px;
+  }
+}
+@media (max-width: 1200px) {
+  .regularPlayButton,
+  .activeButton,
+  .rootButton,
+  .activeRootButton {
+    width: fit-content;
+    padding: 10px 2px;
+  }
+}
+@media (max-width: 1050px) {
+  .regularPlayButton,
+  .activeButton,
+  .rootButton,
+  .activeRootButton {
+    width: fit-content;
+    padding: 10px 5px;
+  }
+}
+@media (max-width: 950px) {
+  .regularPlayButton,
+  .activeButton,
+  .rootButton,
+  .activeRootButton {
+    width: 65px;
+    overflow: hidden;
+    padding: 10px 0px;
+  }
+}
+@media (max-width: 820px) {
+  .regularPlayButton,
+  .activeButton,
+  .rootButton,
+  .activeRootButton {
+    width: 55px;
+  }
+}
+@media (max-width: 690px) {
+  .regularPlayButton,
+  .activeButton,
+  .rootButton,
+  .activeRootButton {
+    width: 45px;
+  }
+}
+@media (max-width: 565px) {
+  .regularPlayButton,
+  .activeButton,
+  .rootButton,
+  .activeRootButton {
+    width: 39px;
```

Comment:
It seems like you're relying on manually setting breakpoints with pixel values at quite a few different widths - sometimes breakpoints can't be avoided but you can get around a lot of this by using things like `flexbox` and `grid` along with properties like `max-width` to get the layout to resize itself without the need for manual intervention. If there's a point at which resizing makes the content look stupid or hard to read then you set a breakpoint and change the page layout to something that works.

Here it looks like you've got a single layout that just resizes - this should be largely achievable without media queries.